app/views/shared/_header.html.erb

```diff
@@ -14,7 +14,11 @@
 				<li class="nav__item <%= controller? "pricing" %>"><a href="/pricing" class="nav__link"><span>Plans &amp; Pricing</span></a></li>
 				<li class="nav__item"><a href="https://enterprise.huboard.com" class="nav__link"><span>Enterprise</span></a></li>
 				<li class="nav__item <%= controller? "integrations" %>"><a href="/integrations" class="nav__link"><span>Integrations</span></a></li>
-        <li class="nav__item nav__item--btn <%= controller? "login" %>"><a href="/login" class="nav__btn <%= controller? "login" %>">Sign in</a></li>
+        <li class="nav__item nav__item--btn <%= controller? "login" %>">
+          <a href="/login/public" class="nav__btn <%= controller? "login" %>">
```

Comment:
I'm concerned about linking directly to `/login/public` here. If the user has already authorized private access, this will prompt to modify the authorization to public-only. Then back to add private again when they try to access a private repo.

I really like "Sign in with GitHub" as a shortcut, but I think we need to wait on that change until the larger OAuth changes described in https://github.com/huboard/huboard-web/issues/172#issuecomment-175113758.